detox/src/devices/drivers/SimulatorDriver.js

```diff
@@ -70,10 +70,8 @@ class SimulatorDriver extends IosDriver {
   }
 
   async launchApp(deviceId, bundleId, launchArgs, languageAndLocale) {
-    const _launchArgs = this._transformLaunchArgs(launchArgs);
-
     await this.emitter.emit('beforeLaunchApp', {bundleId, deviceId, launchArgs});
```

Comment:
@noomorph shouldn't you be changing plugins code so they'd adhere to this new way launch args are passed in?